app/assets/stylesheets/urls.scss

```diff
@@ -24,3 +24,8 @@ p {
     color: white;
   }
 }
+
+div.well.sorted-well {
+  background: black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.